schemas/entity.yml

```diff
@@ -62,7 +72,7 @@
       short: Standardized high-level classification of the entity.
       description: >
         A standardized high-level classification of the entity. This provides a normalized way
-        to group similar entities across different providers or systems. Example values: `bucket`, `database`, `container`, `function`, `queue`, `host`, `user`, `application`, `session`, etc.
+        to group similar entities across different providers or systems. Example values: `bucket`, `database`, `container`, `function`, `queue`, `host`, `user`, `application`, `session`, `cloud`, `orchestrator`, etc.
```

Comment:
I would also add here something similar to 
If an entity is nested under a top-level namespace such as `host` or `cloud`, its `type` array should include the corresponding value - for example, `host` or `cloud`.